kiva/tests/agg/test_text.py

```diff
@@ -21,6 +21,9 @@ def locale_context(category, new=None):
     """
     old = locale.getlocale(category)
     try:
+        # check that we can set the current locale to ensure finally will work
```

Comment:
"to ensure finally" - it looks like something is missing here.

Reply:
This comment and the line it refers to have now been removed.